test/Microsoft.AspNetCore.Mvc.Core.Test/ControllerBaseTest.cs

```diff
@@ -765,7 +765,7 @@ public void HttpUnauthorized_SetsStatusCode()
             var controller = new TestableController();
 
             // Act
-            var result = controller.HttpUnauthorized();
+            var result = controller.Unauthorized();
```

Comment:
In other renaming PRs, we've updated test names et cetera to match the new method names. But :ok: if you're fine w/ the inconsistencies.